source/draw/RasterBgra.ooc

```diff
@@ -21,7 +21,7 @@ RasterBgraCanvas: class extends RasterPackedCanvas {
 	_drawPoint: override func (x, y: Int) {
 		position := this _map(IntPoint2D new(x, y))
 		if (this target isValidIn(position x, position y))
-			this target[position x, position y] = this target[position x, position y] blend(this pen alphaAsFloat, this pen color)
+			this target[position x, position y] = this target[position x, position y] blend(this pen alphaAsFloat, this pen color toBgra())
```

Comment:
It seems like Git is getting smarter in detecting copied code since this recent change was also applied to RasterRgba that does not exist until the following PR.